### PR TITLE
implement theme switcher

### PR DESCRIPTION
**Pull request by BykhovDenis**

### CODE REVIEW

File: packages/ui/src/components/internal/ThemeSelector.svelte [hunk 1]
```diff
@@ -1,28 +1,34 @@
...
+        current = Themes.GREY
+        break;
+      case Themes.GREY:
+        current = Themes.DARK
+        break;
+      case Themes.DARK:
+        current = Themes.LIGHT
+        break;
+    }
+    setTheme(current)
```
Comment (z2sx) [inline packages/ui/src/components/internal/ThemeSelector.svelte:20]: This looks to be overkill :) why not `setTheme(themes[current % themes.length])` where `themes = ['light', 'dark',...]`

**Review by z2sx — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/internal/ThemeSelector.svelte (hunk 1 above)
Comment (BykhovDenis) [reply packages/ui/src/components/internal/ThemeSelector.svelte:20]: Some overengineering, I think in future we need changing icons (when we will have them), and store user setting

Comment (z2sx) [reply packages/ui/src/components/internal/ThemeSelector.svelte:20]: So? :)

Comment (z2sx) [reply packages/ui/src/components/internal/ThemeSelector.svelte:20]: Ping @BykhovDenis

**Review by z2sx — APPROVED** [review]

**Pull request merged**